TASVideos/Pages/Account/Register.cshtml

```diff
@@ -49,6 +49,10 @@
 				</div>
 				<span asp-validation-for="Coppa" class="text-danger"></span>
 			</form-group>
+			<form-group>
+				Also make sure you've read the <a href="/SiteRules">Site Rules</a>.<br/>
+				Your data will be stored according to our <a href="/System/PrivacyPolicy">Privacy Policy</a>.
```

Comment:
Isn't it weird to link to a System page that clearly says "not a page intended for standalone viewing"?

Reply:
I've just copied the login page:
https://github.com/TASVideos/tasvideos/blob/ba987957384a1087e00159f9a6a8865b6789da75/TASVideos/Pages/Account/Login.cshtml#L34
I agree the warning could confuse new users.